packages/salesforcedx-vscode-soql/src/editor/soqlUtils.ts

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { ModelDeserializer, ModelSerializer } from '@salesforce/soql-model';
+import { Impl, Soql, SoqlModelUtils } from '@salesforce/soql-model/lib';
+import { JsonMap } from '@salesforce/ts-types';
+
+export interface ToolingModelJson extends JsonMap {
+  sObject: string;
+  fields: string[];
+}
+
+export class SoqlUtils {
+  public static convertSoqlToUiModel(soql: string): ToolingModelJson {
+    const queryModel = new ModelDeserializer(soql).deserialize();
+    const uimodel = SoqlUtils.convertSoqlModelToUiModel(queryModel);
+    return uimodel;
+  }
+
+  protected static convertSoqlModelToUiModel(
+    queryModel: Soql.Query
+  ): ToolingModelJson {
+    const fields =
+      queryModel.select &&
+      (queryModel.select as Soql.SelectExprs).selectExpressions
+        ? (queryModel.select as Soql.SelectExprs).selectExpressions
+            .filter(expr => !SoqlModelUtils.containsUnmodeledSyntax(expr))
+            .map(expr => ((expr as unknown) as Soql.FieldRef).fieldName)
+        : undefined;
+    const sObject = queryModel.from ? queryModel.from.sobjectName : undefined;
```

Comment:
```suggestion
    const sObject = queryModel.from?.sobjectName;
```

Reply:
Interestingly, I had to shut prettier off on this line after the change.  Have you seen it complain before about using the elvis operator?

Reply:
@dehru Most likely an update to prettier version would solve it